tests/unit/test_vcs.py

```diff
@@ -105,6 +105,7 @@ def test_looks_like_hash():
     assert looks_like_hash(18 * 'a' + '0123456789abcdefABCDEF')
     assert not looks_like_hash(40 * 'g')
     assert not looks_like_hash(39 * 'a')
+    assert not looks_like_hash(41 * 'a')
```

Comment:
Incidental observation: it would be nice to convert this to use `@pytest.mark.parametrize()` in a future commit / PR (not needed for this one).